pkg/sfu/receiver.go

```diff
@@ -380,7 +380,11 @@ func (w *WebRTCReceiver) sendRTCP(packets []rtcp.Packet) {
 		return
 	}
 
-	w.rtcpCh <- packets
+	select {
+	case w.rtcpCh <- packets:
+	default:
+		w.logger.Warnw("sendRTCP failed, rtcp channel full", nil)
```

Comment:
Is this only for `full` condition or `closed` also? Maybe change log to say `full or closed`?

Reply:
no send to a closed channel will cause panic, default is only for full.
in golang principle, need avoid close channel on receive side, since we don't know whether sender will continue send data to it